backend/api/db/models/structure.py

```diff
@@ -22,3 +31,88 @@ class Structure(StructureInsert):
     id: UUID
     created_at: datetime
     updated_at: datetime
+
+
+class StructureInputRow(BaseModel):
+    name: str = Field(alias="Nom*")
+    short_desc: str | None = Field(alias="Description")
+    phone: str | None = Field(alias="Téléphones")
+    address1: str | None = Field(alias="Adresse")
+    address2: str | None = Field(alias="Adresse (complément)")
+    postal_code: str = Field(alias="Code postal*")
+    city: str = Field(alias="Ville*")
+    website: HttpUrl | None = Field(alias="Site web")
+    email: EmailStr | None = Field(alias="Courriel")
+    siret: str | None = Field(alias="Siret")
+    admin_email: EmailStr = Field(alias="Courriel responsable*")
+    admin_firstname: str | None = Field(alias="Prénom responsable")
+    admin_lastname: str | None = Field(alias="Nom responsable")
+    admin_phone_number: str | None = Field(alias="Téléphones responsable")
+    admin_position: str | None = Field(alias="Fonction responsable")
+
+    class Config:
+        anystr_strip_whitespace = True
+
+    @validator("siret", allow_reuse=True)
+    def must_be_a_valid_siret(cls, value: str) -> str:
+        if not value:
+            return value
+        if not validate_luhn(value):
+            raise ValueError(f"{value} not a valid siret")
+        return value
```

Comment:
Est-ce que `validate_luhn` retourne une erreur si `value` est `false` ?
Si ce n'est pas le cas pourquoi le `if not value` avant ?

Ou alors simplifer en ` if value and not validate_luhn(value):`

Reply:
effectivement on ne veut valider que si il y a une valeur (sinon ca rend le champ obligatoire)